cookbooks/bcpc-hadoop/recipes/datanode.rb

```diff
@@ -18,6 +18,21 @@
   end
 end
 
+user_ulimit "hdfs" do
```

Comment:
Is `user_ulimit` a built-in resource in Chef, or does it require us to download a user_ulimit cookbook?